src/tools/log-parser/VERSION

```diff
@@ -0,0 +1 @@
+0.0.1
```

Comment:
IMO the log-parser version should be the same as of kata containers. If you agree then have a look at the runtime's VERSION file which is simply a link to the project's VERSION file.

BTW, I noticed that other tools `agent-ctl` and `trace-forwarder` have their own versions as well. IMO they should all, likewise, have the same version as the project.

What's your opinion? @jodh-intel @kata-containers/architecture-committee

Reply:
@wainersm - agreed.

Reply:
Yes @snir911, you're right regarding handle that in a separate PR. I just created a issue so that we discuss further this idea: https://github.com/kata-containers/kata-containers/issues/4227